Sources/Sentry/include/SentryDemangler.h

```diff
@@ -0,0 +1,22 @@
+#pragma clang diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-parameter"
+#pragma GCC diagnostic ignored "-Wshorten-64-to-32"
+#pragma GCC diagnostic ignored "-Wshadow"
```

Comment:
`l`: Do we really need these here?
```suggestion
```

Reply:
Yes, because of SPM and CocoaPod, they dont have a configuration per file.
If your project is configured to fail with warnings, it will fail.